assets/src/edit-story/components/keyboard/index.js

```diff
@@ -87,6 +87,21 @@ function useKeyEffect(
   );
 }
 
+/**
+ * @param {Node|{current: Node}} refOrNode
+ * @param {string|Array|Object} keyNameOrSpec
+ * @param {function(KeyboardEvent)} callback
+ * @param {Array|undefined} deps
+ */
+export function useKeyEffect(
```

Comment:
This does the exact same thing as the existing `useKeyDownEffect` below.

Reply:
My bad, Updated to pass 'undefined' instead of 'keydown'.

Reply:
To clarify: this is basically the same as `keydown`, but more automatic? I.e. why would we use a "keypress" instead of "keydown"? Which keys prefer this? What's the overall benefit? It also looks like you're using it for arrow keys? Do those keys need "keydown" or "keypress"?

Reply:
[The docs](https://craig.is/killing/mice) say:

> There is a third argument you can use to specify the type of event to listen for. It can be `keypress`, `keydown` or `keyup`.
> 
> It is recommended that you leave this argument out if you are unsure. Mousetrap will look at the keys you are binding and determine whether it should default to `keypress` or `keydown`.

The code is here:

https://github.com/ccampbell/mousetrap/blob/2f9a476ba6158ba69763e4fcf914966cc72ef433/mousetrap.js#L30-L62

If a key is in that list, `keydown` will be used; otherwise `keypress` will be used.

Reply:
Ok, so this is only keydown-vs-keypress then. It sounds like we may want to drop keydown version eventually. But let's keep for now until we know better. I was always under impression that `keypress` is outdated and useless. But I guess that's not the case.